apps/verbs-ui/src/components/Terminal.tsx

```diff
@@ -75,7 +79,7 @@ const Terminal = () => {
       {
         id: 'welcome-8',
         type: 'output',
-        content: '    Web2 -> Web3 library for the OP Stack',
+        content: '   Money Verbs library for the OP Stack',
```

Comment:
Not super tied to this in either direction but I don't know that this needs the "Money" -- just "Verbs library" seems fine to me.

Reply:
```suggestion
        content: '   Verbs library for the OP Stack',
```
Makes sense